docs/en/stack/monitoring/troubleshooting.asciidoc

```diff
@@ -5,4 +5,4 @@
 ++++
 
 See
-{logstash-ref}/monitoring-troubleshooting.html[Troubleshooting {monitoring} in Logstash].
+{logstash-ref}/monitoring-troubleshooting.html[Troubleshooting monitoring in Logstash].
```

Comment:
Unrelated to the beta removal changes but further up in this PR you replaced `{monitoring}` with `{stack-monitor-features}`. Should the same replacement be done here?

Reply:
I figured leaving the simple "monitoring" verb made more sense here.